ci/kind/kind-setup.sh

```diff
@@ -728,5 +758,10 @@ if [[ $ACTION == "create" ]]; then
         echoerr "Only one of '--subnets' and '--extra-networks' can be specified"
         exit 1
     fi
+
+   # Reserve IPs after 192.168.240.63 for e2e tests.
+    if [[ $FLEXIBLE_IPAM == true ]]; then
+        docker network create -d bridge --subnet 192.168.240.0/24 --gateway 192.168.240.1 --ip-range 192.168.240.0/26 kind
```

Comment:
I realized that this will prevent running multiple jobs in parallel if one of them is for FlexibleIPAM. Probably not an issue, but something to keep in mind. cc @jainpulkit22